src/main/scala/mesosphere/marathon/state/LinuxInfo.scala

```diff
@@ -13,30 +14,106 @@ import com.wix.accord._
 case class Seccomp(profileName: Option[String], unconfined: Boolean)
 
 /**
-  * Defines the linux runtime the container will run under.   This could include things such as seccomp, linux capabilities,
+  * Defines the IPC configuration for the instance or executor this is associated with.
+  *
+  * @param ipcMode The IPC mode configured
+  * @param shmSize The shared memory size in MB when using private IPC mode
+  */
+case class IPCInfo(ipcMode: state.IpcMode, shmSize: Option[Int])
+
+/**
+  * Defines the linux runtime the container will run under. This could include things such as seccomp, linux capabilities,
   * and shared pid namespaces.
   *
   * @param seccomp The seccomp mode to use, either unconfined or which profile.
+  * @param ipcInfo The IPCConfig to use
```

Comment:
```suggestion
  * @param ipcInfo The IPC configuration to use
```

Reply:
`IPCConfig` doesn't exist as a type … this could be _the ipcInfo to use_, but overall the doc is not adding much information.